kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -429,6 +429,79 @@ def submit(cls, host, body):
         return submit(host, cls, body)
 
 
+class Sample:
```

Comment:
Maybe we should add a class docstring that explains `BIOSPECIMEN_GROUP` was a previously existing concept that has been used in ingest packages and that's why some fields (external_id) in the Biospecimen and/or Sample class first extract from `SAMPLE` and then secondarily extract from `BIOSPECIMEN_GROUP`